docs/8-changestorage.md

```diff
@@ -0,0 +1,25 @@
+# Change storage
+It is possible to replace the default storage which is a SQLite database by the system of your choice. 
+To do this, you need to create a new type of ORM.
```

Comment:
```suggestion
To do this, you can configure the ORM (TypeOrm) to connect to a different type of database (see https://typeorm.io/#/connection-options/common-connection-options).
However, it may be necessary to change the column types in the entities for a different database type (see https://typeorm.io/#/entities/column-types).
```